### ISSUE TITLE
Closing labuildingsimport.com tasking manager

### ISSUE THREAD

**Issue by maning** [osmlab/labuildings#119]
![screen shot 2017-04-10 at 17 05 29](https://cloud.githubusercontent.com/assets/353700/24859928/01f9668c-1e10-11e7-8fd9-2729d2e441ea.png)
 Per ^^^, we will shutdown soon.  Is there anything we should archive for posterity?

@jschleuss @cityhubla @almccon @batpad @geohacker

**Comment by almccon** [comment]
Looks like the Internet Archive crawled most of the pages, so it's already kinda archived there: https://web-beta.archive.org/web/20161027071239/http://labuildingsimport.com/project/12

**Comment by jschleuss** [comment]
Fine by me to shut down. Thanks for hosting!! @maning @batpad

**Comment by jschleuss** [comment]
I just realized we still need to import Pasadena's buildings, which we excluded because the city split buildings by parts. The last I can recall we were planning on dissolving the buildings if they were touching. So, if we can, should we keep the domain and tasking manager around a bit longer?

**Comment by maning** [comment]
> So, if we can, should we keep the domain and tasking manager around a bit longer?

@jschleuss The feature that we used for labuilding is now part of the US Tasking Manager, as such, I don't see the need of maintaining a separate TM.  If we decide to import Pasadena (I think we should) we can use the US TM instance.

**Comment by batpad** [comment]
I would really appreciate being able to stop supporting labuildingsimport.com - this was always a bit of an in-between solution while we did not have the key feature we needed for imports in tasking manager US.

I am happy to continue maintaining something like a single static HTML page to live on the domain that can contain text / links, etc. so there is continuity with the URL.

Since some of this initial setup happened in a flurry, the domain name is booked under my personal domain account and I recently renewed it for another year, so it's valid another 11 months or so - I'm super happy to initiate a transfer if anyone wishes to take on ownership.

Of course, will make sure to have a check-list before we shut-down and generate backups of the database, etc.

Does this sound good to everyone? If there are strong reasons to keep the Tasking Manager instance around, am super happy to consider, but otherwise I'd propose making a simple static HTML page that explains the project and provides relevant links, hosting new tasks on the US Tasking Manager, and retiring this instance. Thanks everyone!!
